### ISSUE TITLE
AP has issues processing models with history and nGons. Test case assets attached.

### ISSUE THREAD

**Issue by shawstar** [o3de/o3de#9309]
**Describe the bug**
AP has issues processing models with history and nGons. History in FBX is from Maya DCCs tool. Will add a few test cases to this ticket. 

**Steps to reproduce**

**Test case One: SimpleSkinedMesh_with_history_and_ngons.fbx**
This is a .fbx from Maya, this is a cylinder that has bones skinned to it with animation. There is history in the mesh from taking the top of the cylender face of and replacing it will a fill, making the top an nGon.
When the o3de AP sees the mesh, it will import it without an issue:
![image](https://user-images.githubusercontent.com/87207603/165971227-edafa184-c621-4510-ad4e-666dcd2fad74.png)
The file is not usable in o3de, it is a dead file.
![image](https://user-images.githubusercontent.com/87207603/165971338-2c5527d5-1d5f-48bf-968b-66cab58fa580.png)

**Test case Two: SimpleSkinedMesh_with_history.fbx**
This is a .fbx from Maya, this is a cylinder that has bones skinned to it with animation. There is history in the mesh from taking the top of the cylender face of and replacing it will a fill, making the top an nGon, but now we select the nGon and Triangle the nGon.
**Test case Two: SimpleSkinedMesh_with_history.fbx**
![image](https://user-images.githubusercontent.com/87207603/165971773-834b9ae3-ffbc-4232-903b-d5794d7f7e0c.png)
The file is not usable in o3de, it is a dead file.
![image](https://user-images.githubusercontent.com/87207603/165972365-0689227e-b567-4913-9e9f-a57064a7f236.png)

**Test case Three: SimpleSkinedMesh_with_non_deform_history_removed.fbx**
This is a .fbx from Maya, this is a cylinder that has bones skinned to it with animation. There is history in the mesh from taking the top of the cylender face of and replacing it will a fill, making the top an nGon, but now we select the nGon and Triangle the nGon, then the non-deform history was removed. This resulted in the AP throwing a warning.
![image](https://user-images.githubusercontent.com/87207603/165973375-27540053-9b9b-420e-a828-4ce4b548fe38.png)
![image](https://user-images.githubusercontent.com/87207603/165973749-351262bb-b1d7-4c67-b7fb-dad0f736352f.png)
However this assets is actually usable in o3de
![image](https://user-images.githubusercontent.com/87207603/165974069-1a9b2b8f-926c-44e2-b00f-a3812ab453c4.png)
![image](https://user-images.githubusercontent.com/87207603/165974370-6e225da4-dbd7-441d-8d06-044d4ba97a71.png)
The asset looks and animates correctly.

**Test case Four: SimpleSkinedMesh_with_ngons_non_deform_history_removed.fbx**
This is a .fbx from Maya, this is a cylinder that has bones skinned to it with animation. There is history in the mesh from taking the top of the cylender face of and replacing it will a fill, making the top an nGon, but now we select the nGon, then non-deform history was removed. This resulted in the AP throwing a warning.
![image](https://user-images.githubusercontent.com/87207603/165975307-81366097-c74b-49eb-9398-0ec7bab2ffe6.png)
![image](https://user-images.githubusercontent.com/87207603/165975359-8458d7ec-e71f-4b77-bd19-9d997db62040.png)
However this assets is actually usable in o3de
![image](https://user-images.githubusercontent.com/87207603/165975773-328370e6-cd1a-47d8-8b1b-3831ab4a4cdb.png)
![image](https://user-images.githubusercontent.com/87207603/165975904-a87f2204-b2d5-4a3e-bab5-50692d8a3a6e.png)
The asset looks and animates correctly.

**Expected behavior**
Better issue reporting or ignore history

**Actual behavior**
History seems to be the issue, as the nGons did not seem to break anything.

**Assets required**
Provide sample assets needed to reproduce the issue.
[SimpleSkinedMesh_Samples.zip](https://github.com/o3de/o3de/files/8591941/SimpleSkinedMesh_Samples.zip)

**Found in Branch**
Nightly as of 4/28/2022

**Desktop/Device (please complete the following information):**
 - PC,
 - OS: Windows
 - Version 10
 - CPU Intel I9-9900k
 - GPU  NVidia RTX 2080ti
 - Memory [e.g. 16GB]

**Additional context**

**Comment by lmbr-pip** [comment]
Standard practice to freeze transforms and delete history before exporting to FBX? Does this meet the bar for critical?

**Comment by shawstar** [comment]
Critical was decided in the meeting among everyone. Sure, this is standard practice. But also good intentions that might not always be remembered... and or different level of customer experience. I would go to say if this was a standard practice, then there also should be clear warnings if the standard that was not followed to be reported to user.

**Comment by amzn-rhhong** [comment]
@shawstar thanks for the assets! I'm looking into it right now

**Comment by hultonha** [comment]
@shawstar Would you please be able to give an update on this issue. Is this something that's still impacting you day to day? Is this still a critical right now and does the PR @amzn-rhhong merged (#12587) resolve this for you?

**Comment by shawstar** [comment]
Repeating the same test in O3DE, and Other game engines. I found that in all engines tested, you will need to clear the history or issues will happen blocking a successful import.  That said once I did clear non-deform history, just throwing together a similar test, the results were good in both game engines. Including O3DE. 
![image](https://user-images.githubusercontent.com/87207603/200621765-7cbc6927-4d0a-4805-af56-e8fbaed717e3.png)

[Dirty_Model_Test.zip](https://github.com/o3de/o3de/files/9963238/Dirty_Model_Test.zip)

I think this issue is okay to close.
